packages/react-components/react-carousel-preview/src/components/CarouselNavImageButton/CarouselNavImageButton.types.ts

```diff
@@ -12,6 +20,9 @@ export type CarouselNavImageButtonProps = ComponentProps<CarouselNavImageButtonS
 /**
  * State used in rendering CarouselNavImageButton
  */
-export type CarouselNavImageButtonState = ComponentState<CarouselNavImageButtonSlots>;
-// TODO: Remove semicolon from previous line, uncomment next line, and provide union of props to pick from CarouselNavImageButtonProps.
-// & Required<Pick<CarouselNavImageButtonProps, 'propName'>>
+export type CarouselNavImageButtonState = ComponentState<CarouselNavImageButtonSlots> & {
+  /**
+   * Enables selection state control
+   */
+  isSelected?: boolean;
```

Comment:
```suggestion
  selected?: boolean;
```

nit: we don't use `is` prefix on `Props` & `State` objects